src/main/deltachat.js

```diff
@@ -442,4 +442,59 @@ class DeltaChatController {
   }
 }
 
+function snakeCaseKeys (obj) {
```

Comment:
seems like this could be done on the frontend -- `<input>` accepts underscore variables, so you would then only need to translate the security flags to sever flags.

but if you think it's fine, feel free to merge it and we could simplify it later.